src/swarm-scenario/Swarm/Game/Tick.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+-- |
+-- SPDX-License-Identifier: BSD-3-Clause
+module Swarm.Game.Tick (
```

Comment:
Why is this module needed in `swarm-scenario` (as opposed to `swarm-engine`)?

Reply:
It may not be; I've added it to the follow-up list in the PR description.